test/StakeManager.js

```diff
@@ -331,6 +403,32 @@ describe('StakeManager', function () {
       assertBNEqual(staker.stake, stake, 'Stake should not change');
     });
 
+    it('Staker should not be able to withdraw if committed during lock period despite not revealing', async function () {
+      let epoch = await getEpoch();
+      const amount = tokenAmount('10000');
+      const stakerId = stakeManager.stakerIds(signers[2].address);
```

Comment:
like this

Reply:
I checked the file and added await in 2 lines and checked file rest of the await is there where it should be, but the warning is still showing particularly only for this test case https://github.com/razor-network/contracts/pull/312/files#diff-0ea851f7ce30fd6fc4b4e3c5babae84457d03368116d3c1d4212c2b04cb2a575R559
await is there and also after adding await at two more places the same warning for same particular test case is still there don't know exactly why is this happening.

Reply:
hey @GauravJain9  is this solved ?

Reply:
yes it is solved @abhishekvispute will update

Reply:
Done and updated @abhishekvispute